phpseclib/Net/SSH2.php

```diff
@@ -867,6 +867,14 @@ class SSH2
     var $agent;
 
     /**
+     * Connection storage to imulate original wrapper behvior:
```

Comment:
emulate? behvior?

I don't think a reference such as "the original haviour" is useful.